bundles/org.eclipse.equinox.common/src/org/eclipse/core/runtime/IProgressMonitor.java

```diff
@@ -253,6 +253,37 @@ public default IProgressMonitor slice(int work) {
 		return new SlicedProgressMonitor(this, work);
 	}
 
+	/**
+	 * Creates a new IProgressMonitor that will consume the given number of ticks
+	 * from this IProgressMonitor.
+	 * 
+	 * @param totalWork number of ticks to consume from the receiver
+	 * @parma options optional ProgressOption
+	 * @return new IProgressMonitor that may be used in place of a new
+	 *         IProgressMonitor
+	 * @since 3.19
+	 */
+	public default IProgressMonitor create(int totalWork, ProgressOption... options) {
```

Comment:
Create sounds strange here, can we maybe name this simply 
```suggestion
	public default IProgressMonitor submonitor(int totalWork, ProgressOption... options) {
```

Reply:
ok